tools/mmp/driver.cs

```diff
@@ -614,6 +614,18 @@ static void Pack (IList<string> unprocessed)
 					else
 						native_libs.Add (kvp.Key, kvp.Value);
 				}
+				// Idealy, this would be handled by Link () above. However in the non-linking case
+				// we do not run MobileMarkStep which generates the pinvoke list. Hack around this for now
+				// https://bugzilla.xamarin.com/show_bug.cgi?id=43419
+				if (App.LinkMode == LinkMode.None) {
+					foreach (var kvp in ProcessDllImports ()) {
```

Comment:
I could for sure. I leaned this way @spouliot because i'm hoping that when Rolf returns he can take a look at this (this the bug) and fix it more right. Thus a bit of copy paste, but keep the hack self-contained.

He wanted to nuke all the secondary processing and just do everything in Link, so this is a step backwards.

I can change if desired though.